src/translations/de.yml

```diff
@@ -1,25 +1,34 @@
 consentModal:
-  title: Informationen die wir sammeln
+  title: Informationen zur Privatsphäre
   description: >
-    Hier können Sie einsehen und anpassen, welche Information wir über Sie sammeln.
+    Hier können Sie einsehen und anpassen, welche persönlichen Daten gespeichert werden.
   privacyPolicy:
     name: Datenschutzerklärung
     text: >
       Bitte lesen Sie unsere {privacyPolicy} um weitere Details zu erfahren.
 consentNotice:
   changeDescription: Es gab Änderungen seit Ihrem letzten Besuch, bitte aktualisieren Sie Ihre Auswahl.
   description: >
-    Wir speichern und verarbeiten Ihre personenbezogenen Informationen für folgende Zwecke: {purposes}.
+    Wir speichern und verarbeiten Ihre persönlichen Daten für folgende Zwecke: {purposes}.
   learnMore: Mehr erfahren
+accept: Akzeptieren
+acceptTitle: Cookies akzeptieren
+acceptAll: Alle Einstellungen akzeptieren
 save: Speichern
+saveData: Meine Einstellungen speichern
 decline: Ablehnen
+declineAll: Alle Einstellungen ablehnen
+close: Schließen
+enabled: Aktiviert
+disabled: Deaktiviert
 app:
   optOut:
-    title: (Opt-Out)
-    description: Diese Anwendung wird standarmäßig gelanden (aber Sie können sie deaktivieren)
+    title: (optOut)
```

Comment:
Everything seems fine but this change feels a little weird to me (the fact that it is camel-cased in particular).
Are you sure about it?
Would there be a more appropriate german word?

Reply:
Indeed, information I got back is that Opt-Out is the most used term. Same thing for the Estonian translation. I updated both PR.